lib/bun_bo.rb

```diff
@@ -16,7 +16,7 @@ def run(input_path)
       FileUtils.mkdir_p(test_folder)
       test_path.write('')
     else
-      puts "File does not exists"
+      "File does not exists"
```

Comment:
If you returns the value here, you should also change the code in `exec/bun_bo`. Another solution is to raise an error here.

Reply:
Returned code to origin